pkg/rtc/participant.go

```diff
@@ -672,9 +673,9 @@ func (p *ParticipantImpl) Close(sendLeave bool) error {
 	return nil
 }
 
-func (p *ParticipantImpl) Negotiate() {
+func (p *ParticipantImpl) Negotiate(force bool) {
```

Comment:
a bit of comment describing what force does would be helpful for others reading the code.